### DISCUSSION TITLE
In-Page Anchors & Linking?

### DISCUSSION THREAD

**Discussion by mbluett88** [zim-desktop-wiki/zim-desktop-wiki#2636] [Ideas]
I have managed to implement an in-page anchor and then linking to it from another page. However, I find the implementation difficult to use.

The main problem with this is that the Link dialog shows the link as a text file rather than using the colon-hierarchical representation. 

This forces a user to manually manufacture the colon-hierarchy and then to add a # symbol followed by the in-page anchor name. Although this is possible to do, it is a poor implementation and not very obvious, especially to new users.

I would, therefore, suggest that this be given a priority to fix and thus making it easier to implement a link to an in-page anchor, which I would speculate does not get used much.

At a minimum, the documentation should be updated to show how to link to an in-page anchor.

**Comment by jaap-karssenberg** [comment]
Indeed, using the "browse" button you get a file link instead of a page
link - this is open to improvement from usability point of view

To get a page link, just start typing and the completion will suggest
matching pages.

--- Jaap
